### ISSUE TITLE
Prediction in train mode?

### ISSUE THREAD

**Issue by ChuhuaW** [zorzi-s/PolyWorldPretrainedNetwork#3]
Hi,

Thank you for releasing the code. I'm wondering why you set the network to train() mode during inferencing? Thanks.

```
model = R2U_Net()
model = model.cuda()
model = model.train()

head_ver = DetectionBranch()
head_ver = head_ver.cuda()
head_ver = head_ver.train()

suppression = NonMaxSuppression()
suppression = suppression.cuda()

matching = OptimalMatching()
matching = matching.cuda()
matching = matching.train()
```

**Comment by zorzi-s** [comment]
Hi @ChuhuaW,

Thank you for the interest in our project! 
The model is set to train() mode during inference to make sure that the BatchNorm layers rely on batch statistics rather than the mean and variance estimated during training. Experimentally, using batch stats makes the network perform better during inference.

**Comment by ChuhuaW** [comment]
> Hi @ChuhuaW,
> 
> Thank you for the interest in our project! The model is set to train() mode during inference to make sure that the BatchNorm layers rely on batch statistics rather than the mean and variance estimated during training. Experimentally, using batch stats makes the network perform better during inference.

Thanks for the reply @zorzi-s. I figured that out by setting the running statistics to None. Closing the issue now.
